pages/_app.tsx

```diff
@@ -19,17 +43,27 @@ export default class MyApp extends App {
 
 	render(): JSX.Element {
 		const { Component, pageProps } = this.props
+		const theme = MakeTheme(this.state.themeType)
 
 		return (
 			<>
 				<Head>
 					{/* https://err.sh/next.js/no-document-title */}
-					<title>My new cool app</title>
+					<title>Impact</title>
 				</Head>
 				<ThemeProvider theme={theme}>
 					{/* CssBaseline kickstart an elegant, consistent, and simple baseline to build upon. */}
 					<CssBaseline />
-					<Component {...pageProps} />
+					<AppContext.Provider
+						value={{
+							themeType: this.state.themeType,
+							setThemeType: this.setThemeType,
+							themePrimary: this.state.primary,
+							themeSecondary: this.state.secondary,
```

Comment:
Once again, are `primary` and `secondary` doing anything or can they be removed?